### ISSUE TITLE
Axe linter 4.10.1 on windows errors on start-up

### ISSUE THREAD

**Issue by hamirmahal** [dequelabs/axe-core#4644]
### Product

axe Extension

### Question

The VSCode extension repeatedly crashes after a recent update.

**Comment by hamirmahal** [comment]
![Image](https://github.com/user-attachments/assets/f0afee67-5f9f-465c-ad8b-5493296b8236)

**Comment by hamirmahal** [comment]
This is from opening up [`sdk-go`](https://github.com/restatedev/sdk-go/)`/.github/workflows`.

**Comment by hamirmahal** [comment]
Clicking `Repository` for the `axe Accessibility Linter` extension in VSCode takes me to https://github.com/dequelabs.

**Comment by hamirmahal** [comment]
https://github.com/user-attachments/assets/69595bfd-70d9-4cc8-826a-85fa3d166e02

**Comment by hamirmahal** [comment]
![Image](https://github.com/user-attachments/assets/f0afee67-5f9f-465c-ad8b-5493296b8236)

**Comment by hamirmahal** [comment]
```
[Info  - 4:07:54 PM] Connection to server got closed. Server will restart.
/home/hamir/.vscode/extensions/deque-systems.vscode-axe-linter-4.10.1/dist/bin/LSP-server-4.10.1: 1: ELF��n�@: not found
/home/hamir/.vscode/extensions/deque-systems.vscode-axe-linter-4.10.1/dist/bin/LSP-server-4.10.1: 2: Syntax error: Unterminated quoted string
[Error - 4:07:54 PM] Connection to server is erroring. Shutting down server.
/home/hamir/.vscode/extensions/deque-systems.vscode-axe-linter-4.10.1/dist/bin/LSP-server-4.10.1: 1: ELF��n�@: not found
/home/hamir/.vscode/extensions/deque-systems.vscode-axe-linter-4.10.1/dist/bin/LSP-server-4.10.1: 2: Syntax error: Unterminated quoted string
```

**Comment by hamirmahal** [comment]
I already tried removing the extension, and then reinstalling it. I still got the same error almost immediately.

**Comment by WilcoFiers** [comment]
We are working on the problem. Apologies for the inconvenience.

**Comment by WilcoFiers** [comment]
We think we've identified the issue, but so close to US holidays we didn't want to chance it. We've reverted to axe linter 4.10.0 and have re-released it axe 4.10.2. **Please upgrade to 4.10.2 to solve this issue**. That should solve any issues. We'll work on getting the changes promised in 4.10.1 released next week and will get these out as soon as possible. We'll also be sure to tighten up the release process so that this can never happen again. Our apologies for any inconvenience.

If anyone does still have issues with the latest install (can't think why, but who knows), please let us know in this ticket.
